Tests fails on s390x due to low thread stack size
upstream report: https://github.com/Perl/perl5/issues/18160

diff --git a/t/thread_it.pl b/t/thread_it.pl
index f38a580..e8c450a 100644
--- a/t/thread_it.pl
+++ b/t/thread_it.pl
@@ -36,7 +36,7 @@ my $curr = threads->create({
                             stack_size => $^O eq 'hpux'   ? 524288 :
                                           $^O eq 'darwin' ? 2000000:
                                           $^O eq 'VMS'    ? 150000 :
-                                          $^O eq 'aix'    ? 1500000 : 0,
+                                          $^O eq 'aix'    ? 1500000 : 256*1024,
                            }, sub {
 			       run_tests();
 			       return defined &curr_test ? curr_test() : ()
